apps/wakunode2/config.nim

```diff
@@ -207,11 +205,16 @@ type
       defaultValue: false
       name: "keep-alive" }: bool
 
-    topics* {.
-      desc: "Default topics to subscribe to (space separated list)."
-      defaultValue: "/waku/2/default-waku/proto"
+    topicsDeprecated* {.
+      desc: "Default topics to subscribe to (space separated list). DEPRECATED: please use repeated --topic argument instead."
+      defaultValue: ""
```

Comment:
should we keep the `/waku/2/default-waku/proto` here? otherwise old deployments using "topics" will break?

Reply:
Oh! Important point that I missed. @alrevuelta is right here. I think we should keep the default in both config options.

Reply:
It shouldn't break because if nothing is specified, the default pubsub topic will get used.

Reply:
you are right:
https://github.com/waku-org/nwaku/blob/493f082975e7b1bef231acf0299453d645388b8e/apps/wakunode2/app.nim#L559

but perhaps a bit convoluted? can see you applied the changes thanks!